draft-ietf-scitt-architecture.md

```diff
@@ -392,7 +392,7 @@ Registration Policies refer to additional checks over and above the Mandatory Re
 
 Transparency Services MUST maintain Registration Policies.
 
-Transparency Services MUST also maintain a list of trust anchors used to authenticate Issuers, which MAY be included in a registration policy statement.
+Transparency Services MUST also maintain a list of trust anchors, which MAY be used by Relying Parties to authenticate Issuers, and which MAY be included in a registration policy statement.
```

Comment:
Why not should? What happens if this guidance is ignored?

Reply:
Yes, I think SHOULD is better

Reply:
```suggestion
Transparency Services MUST also maintain a list of trust anchors, which SHOULD be used by Relying Parties to authenticate Issuers, and which MAY be included in a registration policy statement.
```